### DISCUSSION TITLE
How to test/debug i18n

### DISCUSSION THREAD

**Discussion by topofocus** [bridgetownrb/bridgetown#571] [General]
Hi,
I just set up two locales according to the official introduction, set default_locale to `de` and tried to put an entry in the bulma-starter-template into that regime.

Created an entry in both `de.yml` and `en.yml`

The exchange of {% t recent_articles %} worked. 

Unfortunately, it worked in the wrong direction.  `site.locale`is `:de`, but it displays the translation of the english dictionary. 
The log shows: the first request is `GET /de `, thus the language seems to be recognized. 

How to debug this accident?

**Comment by jaredcwhite** [comment]
Hi @topofocus, I'm unable to reproduce your issue. It's translating correctly for me even when the default_locale is something other than `en`. Are you able to put a test repo on GitHub I can take a look at?

**Reply by topofocus** [reply]
Ha – created a repo, pushed it to github ... and the error is gone.
Maybe caused by the browser cache.

Therefor I asked for advice how to debug such behavior.

**Reply by jaredcwhite** [reply]
Glad you got it working 👍

**Comment by topofocus** [comment]
Something is strange.

Modified  `bulmatown/buttom` to translate the label.  That seemed very easy, just replace 
`<span> {{ label }} </span>` with  `<span> {{ label  | t }} </span>`

But this did'nt work.

Spend some time to figure out, that I have to change  more then that temporary. 
After adding something in the `index.md`file, the translation was displayed. Before, even a restart of the server did not change the display of the buttom.  I then removed the inserted word and anything works 

Seems to me, that the caching is to restrictive for development.  

`
